src/starkex/proof-splitter.md

```diff
@@ -0,0 +1,65 @@
+Due to the gas limitations when verifying an original STARK proof, the proof generated by the Stone Prover should be split into three types: the main proof, Trace Merkle proof, and FRI proof. Typically, this includes one Main Proof accompanied by several Trace Merkle Proofs and FRI Proofs.
+
+This process depends on the [fact registry](https://zksecurity.github.io/stark-book/starkex/facts.html), which serves as verification snapshots. This allows the Main Proof to be verified based on earlier snapshots obtained from the Merkle Proofs and FRI Proofs, thereby circumventing the bottleneck caused by gas limitations in a single EVM transaction.
```

Comment:
I think you should be able to link to this by just writing something like `./starkex/facts.html` instead of an absolute URL (which might change if we change domain name at some point)

Reply:
maybe a good way to explain these could be that they are subproofs that have been verified already :D